lib/lknvball_web/schema/event_user.ex

```diff
@@ -1,23 +1,21 @@
 defmodule LknvballWeb.Schema.EventUser do
-  # alias LknvballWeb.Resolvers
+  alias Lknvball.Accounts
+  alias Lknvball.Events
 
   use Absinthe.Schema.Notation
   use Absinthe.Relay.Schema.Notation, :modern
 
+  import Absinthe.Resolution.Helpers, only: [dataloader: 1]
+
   connection(node_type: :event_user)
 
   node object(:event_user) do
     field(:paid, :boolean)
 
     field(:event, non_null(:event)) do
-      resolve(fn %{event_id: event_id}, _, _ -> {:ok, Lknvball.Events.get_event!(event_id)} end)
+      resolve(fn %{event_id: event_id}, _, _ -> {:ok, Events.get_event!(event_id)} end)
```

Comment:
Eventually we need to get resolve functions out of schema but this is perfectly fine for now - also second note we need to refactor get_event! and all bang functions to handle errors

This is a non gating comment/fix.

Reply:
I will go ahead and move this actually.  Might do the bang fix too.  Not sure.  If I don't I'll throw a TODO.